src/components/Note/Note.tsx

```diff
@@ -25,6 +26,7 @@ interface NoteProps {
   showAuthors: boolean;
   childrenNotes: Array<NoteClientModel & {authorName: string; votes: VoteClientModel[]}>;
   votes: VoteClientModel[];
+  voteConfiguration: VoteConfigurationClientModel;
```

Comment:
Do you really need the whole configuration or only `showVotesOfOtherUsers`?

Reply:
Removed the voteConfiguration.